src/dialog_style_editor.cpp

```diff
@@ -314,7 +314,8 @@ DialogStyleEditor::DialogStyleEditor(wxWindow *parent, AssStyle *style, agi::Con
 	// Outline
 	add_with_label(OutlineBox, _("Outline:"), Outline);
 	add_with_label(OutlineBox, _("Shadow:"), Shadow);
-	add_with_label(OutlineBox, _("Border style:"), OutlineType);
+	OutlineBox->Add(new wxStaticText(this, -1, _("Border style:")), wxSizerFlags().Center().HorzBorder());
+	OutlineBox->Add(OutlineType, wxSizerFlags().Left().Expand());
```

Comment:
For consistency, I'd rather add an optional parameter `proportion` to `add_with_label`. As far as I can tell, the sizer flags for the label match here match the one in `add_with_label`, right?